### DISCUSSION TITLE
Avery 5267 Label Dimensions?

### DISCUSSION THREAD

**Discussion by ndbnimbus** [snipe/snipe-it#17615] [Q&A]
Does anyone have label dimensions for avery 5267 80 labels per page? we cannot get the left margin to do anything.. 

#### Please confirm you have done the following before posting your bug report:

- [X ] I have enabled debug mode 
- [ X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Server (please complete the following information):**
 - Snipe-IT Version 
 - OS: [e.g. Ubuntu, CentOS]
 - Web Server: [e.g. Apache, IIS]
 - PHP Version 

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Error Messages**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include the output from `php -m` (this should display what modules you have enabled.)

**Additional context**
- Is this a fresh install or an upgrade? 
- What OS and web server you're running Snipe-IT on
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
